README.md

```diff
@@ -1,82 +1,112 @@
+
+![analytics image (flat)](https://raw.githubusercontent.com/vitr/google-analytics-beacon/master/static/badge.svg)
+![analytics](https://www.google-analytics.com/collect?v=1&cid=555&t=pageview&ec=repo&ea=open&dp=/urbanpy/readme&dt=&tid=UA-4677001-16)
+
 **SCL Data - Data Ecosystem Working Group**
 
 [![IDB Logo](https://scldata.iadb.org/assets/iadb-7779368a000004449beca0d4fc6f116cc0617572d549edf2ae491e9a17f63778.png)](https://scldata.iadb.org)
 
+# Welcome to idbsocialdatapy
+ 
+The objective of this library is to help users access standardized SCLdata indicators on topics such as poverty, inequality, health, labor market, gender and diversity, education, and migration for the 26 countries of the region. 
 
-## Table of Contents:
----
+The project to develop the library in Python resulted from the implementation workshop for IDBSocialData. Individual developers showed interest in the library in Python and decided to collaborate voluntarily with the IDB to develop this tool. 
 
-- [Objective and context](#objective-and-context)
-- [Collaboration](#description-and-context)
-- [Road Map](#road-map)
-- [Author/s](#authors)
-- [License](#license)
-- [Limitation of responsibilities](#limitation-of-responsibilities)
+## Installation
 
-## Objective and context
- <font size="1">**Important**: This package is in the development phase</font>
- 
-The objective of this tool is to help users access standardized SCLdata indicators on topics such as poverty, inequality, health, labor market, gender and diversity, education, and migration for the 26 countries of the region.  This tool will be developed to facilitate the consultation of the region's indicators, providing a unified source of data on different topics that can be easily used to generate comparative analyses or studies on the region. This will boost knowledge about the region, facilitating decision-making with a reliable and comparable source of information for the countries of Latin America and the Caribbean. 
+To install the idbsocialpy library you can use:
 
-The project to develop the library in Python resulted from the implementation workshop for IDBSocialData. Individual developers showed interest in the library in Python and decided to collaborate voluntarily with the IDB to develop this tool. 
-At the end of the project, all the developers who participate will be included in the package as authors.
+```sh
+$ pip install idbsocialpy
```

Comment:
I see a mix of the use of  `idbsocialpy` and `idbsocialdatapy`,

Reply:
Perfect, I changed it.